data/sars_cov_2/primer_schemes/ARTIC/SARS-CoV-2/V1/SARS-CoV-2.scheme.fasta

```diff
@@ -1,196 +1,196 @@
-TGTCGCTTCCAAGAAAAGGA
->ARTIC_V1_cropped_primer_seq_92
-AGGACTGGTATGATTTTGTA
->ARTIC_V1_cropped_primer_seq_94
-TGTTGACACTGACTTAACAA
->ARTIC_V1_cropped_primer_seq_96
-AGGAATTACTTGTGTATGCT
->ARTIC_V1_cropped_primer_seq_98
-GTTGATAAGTACTTTGATTG
->ARTIC_V1_cropped_primer_seq_100
-TCAATAGCCGCCACTAGAGG
->ARTIC_V1_cropped_primer_seq_102
-CATCAGGAGATGCCACAACT
->ARTIC_V1_cropped_primer_seq_104
-AGCAAAATGTTGGACTGAGA
->ARTIC_V1_cropped_primer_seq_106
-TGAGTTAACAGGACACATGT
->ARTIC_V1_cropped_primer_seq_108
-ACTCAACTTTACTTAGGAGG
->ARTIC_V1_cropped_primer_seq_110
-ACCTAGACCACCACTTAACC
->ARTIC_V1_cropped_primer_seq_112
-ATTCTACACTCCAGGGACCA
->ARTIC_V1_cropped_primer_seq_114
-TGATTTGAGTGTTGTCAATG
->ARTIC_V1_cropped_primer_seq_116
-TCACGCATGATGTTTCATCT
->ARTIC_V1_cropped_primer_seq_118
-TGATAGAGACCTTTATGACA
->ARTIC_V1_cropped_primer_seq_120
-TGTTTATCACCCGCGAAGAA
->ARTIC_V1_cropped_primer_seq_122
-GGCACATGGCTTTGAGTTGA
->ARTIC_V1_cropped_primer_seq_124
-TGTTAAGCGTGTTGACTGGA
->ARTIC_V1_cropped_primer_seq_126
-TCGATAGATATCCTGCTAAT
->ARTIC_V1_cropped_primer_seq_128
-GCTGGCTTTAGCTTGTGGGT
->ARTIC_V1_cropped_primer_seq_130
-GGGTGTGGACATTGCTGCTA
->ARTIC_V1_cropped_primer_seq_132
-GTTGTCCAACAATTACCTGA
->ARTIC_V1_cropped_primer_seq_134
-ACAGGTTCATCTAAGTGTGT
->ARTIC_V1_cropped_primer_seq_136
-TGTCGCAAAATATACTCAAC
->ARTIC_V1_cropped_primer_seq_138
-ACAAAAGAAAATGACTCTAA
->ARTIC_V1_cropped_primer_seq_140
-ACAAATCCAATTCAGTTGTC
->ARTIC_V1_cropped_primer_seq_142
-ACACGTGGTGTTTATTACCC
->ARTIC_V1_cropped_primer_seq_144
-CAATTTTGTAATGATCCATT
->ARTIC_V1_cropped_primer_seq_146
-ACATCACTAGGTTTCAAACT
->ARTIC_V1_cropped_primer_seq_148
-AGAGTCCAACCAACAGAATC
->ARTIC_V1_cropped_primer_seq_150
-AGGGCAAACTGGAAAGATTG
->ARTIC_V1_cropped_primer_seq_152
-CCAGCAACTGTTTGTGGACC
->ARTIC_V1_cropped_primer_seq_154
-CAACTTACTCCTACTTGGCG
->ARTIC_V1_cropped_primer_seq_156
-GTGGTGATTCAACTGAATGC
->ARTIC_V1_cropped_primer_seq_158
-TTGCCTTGGTGATATTGCTG
->ARTIC_V1_cropped_primer_seq_160
-GCACTTGGAAAACTTCAAGA
->ARTIC_V1_cropped_primer_seq_162
-GGGCTATCATCTTATGTCCT
->ARTIC_V1_cropped_primer_seq_164
-TCCTTTGCAACCTGAATTAG
->ARTIC_V1_cropped_primer_seq_166
-TGCTGTAGTTGTCTCAAGGG
->ARTIC_V1_cropped_primer_seq_168
-ACTAGCACTCTCCAAGGGTG
->ARTIC_V1_cropped_primer_seq_170
-TCAGGTGATGGCACAACAAG
->ARTIC_V1_cropped_primer_seq_172
-CGACTACTAGCGTGCCTTTG
->ARTIC_V1_cropped_primer_seq_174
-CCATGGCAGATTCCAACGGT
->ARTIC_V1_cropped_primer_seq_176
-GTACGCGTTCCATGTGGTCA
->ARTIC_V1_cropped_primer_seq_178
-ACACAGACCATTCCAGTAGC
->ARTIC_V1_cropped_primer_seq_180
-TCACTACCAAGAGTGTGTTA
->ARTIC_V1_cropped_primer_seq_182
-TTTGTGCTTTTTAGCCTTTC
->ARTIC_V1_cropped_primer_seq_184
-TGAGGCTGGTTCTAAATCAC
->ARTIC_V1_cropped_primer_seq_186
-GGCCCCAAGGTTTACCCAAT
->ARTIC_V1_cropped_primer_seq_188
-TGAGGGAGCCTTGAATACAC
->ARTIC_V1_cropped_primer_seq_190
-GCCAACAACAACAAGGCCAA
->ARTIC_V1_cropped_primer_seq_192
-TGGATGACAAAGATCCAAAT
->ARTIC_V1_cropped_primer_seq_194
-AACAATTGCAACAATCCATG
+>ARTIC_V1_primer_seq_0
+ACCAACCAACTTTCGATCTCTTGT
```

Comment:
Changes for files in `data/sars_cov_2/primer_schemes/`:
* primer sequences are no longer cropped to 20b. The exact primer sequence (which has a variable length) is searched
* for each primer scheme, a serialised ahocorasick trie structure containing the sequences is calculated and stored (see pickle file).
* these files are generated using the `fetch_primers.py` script